### PR TITLE
Parse queryTimeout when either string or number

### LINKED ISSUE #440: Exception is thrown when configuring Query Timeout
I have configured the Query Timeout to 3600 seconds and now see an interface conversion exception when clicking 'Test' from the Data Source settings page. 
This is seen on 3.1.0

The traceback in the logs follows: 
```
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149123341Z level=debug msg="panic: interface conversion: interface {} is float64, not string"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149178193Z level=debug msg=
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149188708Z level=debug msg="goroutine 33 [running]:"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149200328Z level=debug msg="github.com/grafana/clickhouse-datasource/pkg/plugin.LoadSettings({0x9, {0xc000136288, 0x11}, {0xc0004cc3c0, 0x1d}, {0xc0004ce598, 0x8}, {0x0, 0x0}, {0x0, ...}, ...})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149212329Z level=debug msg="\t/drone/src/pkg/plugin/settings.go:121 +0x15c5"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149223161Z level=debug msg="github.com/grafana/clickhouse-datasource/pkg/plugin.(*Clickhouse).Connect(0x4f5b01?, {0x9, {0xc000136288, 0x11}, {0xc0004cc3c0, 0x1d}, {0xc0004ce598, 0x8}, {0x0, 0x0}, ...}, ...)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149234479Z level=debug msg="\t/drone/src/pkg/plugin/driver.go:105 +0xd8"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149245912Z level=debug msg="github.com/grafana/sqlds/v2.(*SQLDatasource).NewDatasource(0xc0004b8dc0, {0x9, {0xc000136288, 0x11}, {0xc0004cc3c0, 0x1d}, {0xc0004ce598, 0x8}, {0x0, 0x0}, ...})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149256952Z level=debug msg="\t/go/pkg/mod/github.com/grafana/sqlds/v2@v2.5.0/datasource.go:84 +0x7f"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149294379Z level=debug msg="main.newDatasource({0x9, {0xc000136288, 0x11}, {0xc0004cc3c0, 0x1d}, {0xc0004ce598, 0x8}, {0x0, 0x0}, {0x0, ...}, ...})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149304703Z level=debug msg="\t/drone/src/pkg/main.go:23 +0x78"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149325628Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/backend/datasource.(*instanceProvider).NewInstance(0xc0004960d8?, {0x1, {0xc0004cc3c0, 0x1d}, 0xc000496b00, 0x0, 0xc00001c780})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149337541Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/backend/datasource/instance_provider.go:58 +0xd5"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149362397Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/backend/instancemgmt.(*instanceManager).Get(0xc0004960c0, {0x1, {0xc0004cc3c0, 0x1d}, 0xc000496b00, 0x0, 0xc00001c780})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149373381Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/backend/instancemgmt/instance_manager.go:114 +0x595"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.14938456Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/internal/automanagement.(*Manager).CheckHealth(0x136b5c8?, {0x136b5c8, 0xc000111170}, 0xc000496b40)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149395308Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/internal/automanagement/manager.go:39 +0x8e"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149405061Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/backend.(*diagnosticsSDKAdapter).CheckHealth(0xc00048a040, {0x136b5c8, 0xc000111170}, 0xc0004964c0)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149415855Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/backend/diagnostics_adapter.go:50 +0x1b8"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149425786Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/backend/grpcplugin.(*diagnosticsGRPCServer).CheckHealth(0xc4d913?, {0x136b5c8?, 0xc000111170?}, 0xc000111140?)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149435749Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/backend/grpcplugin/grpc_diagnostics.go:52 +0x2b"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149450483Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/genproto/pluginv2._Diagnostics_CheckHealth_Handler.func1({0x136b5c8, 0xc000111170}, {0x1087ea0?, 0xc0004964c0})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149461435Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/genproto/pluginv2/backend_grpc.pb.go:293 +0x78"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149487366Z level=debug msg="github.com/grpc-ecosystem/go-grpc-prometheus.(*ServerMetrics).UnaryServerInterceptor.func1({0x136b5c8, 0xc000111170}, {0x1087ea0, 0xc0004964c0}, 0xc000167628?, 0xc000124b88)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149499541Z level=debug msg="\t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-prometheus@v1.2.0/server_metrics.go:107 +0x87"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149509378Z level=debug msg="github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func2.1({0x136b5c8?, 0xc000111170?}, {0x1087ea0?, 0xc0004964c0?})"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149519172Z level=debug msg="\t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.4.0/chain.go:48 +0x4f"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149528949Z level=debug msg="go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc.UnaryServerInterceptor.func1({0x136b5c8, 0xc0004dd050}, {0x1087ea0, 0xc0004964c0}, 0xc00048a560, 0xc000496a00)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149538505Z level=debug msg="\t/go/pkg/mod/go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc@v0.40.0/interceptor.go:342 +0x528"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149558384Z level=debug msg="github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func2({0x136b5c8, 0xc0004dd050}, {0x1087ea0, 0xc0004964c0}, 0xc00048a560, 0xfe1400?)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149569922Z level=debug msg="\t/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.4.0/chain.go:53 +0x14e"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149585693Z level=debug msg="github.com/grafana/grafana-plugin-sdk-go/genproto/pluginv2._Diagnostics_CheckHealth_Handler({0xfe8380?, 0xc0004d8930}, {0x136b5c8, 0xc0004dd050}, 0xc0004ec070, 0xc0004dc5d0)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149595406Z level=debug msg="\t/go/pkg/mod/github.com/grafana/grafana-plugin-sdk-go@v0.161.0/genproto/pluginv2/backend_grpc.pb.go:295 +0x138"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149620194Z level=debug msg="google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000e2000, {0x1375718, 0xc0000076c0}, 0xc0003ba6c0, 0xc0004dcba0, 0x1bdcd40, 0x0)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149630637Z level=debug msg="\t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1345 +0xdf3"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149645394Z level=debug msg="google.golang.org/grpc.(*Server).handleStream(0xc0000e2000, {0x1375718, 0xc0000076c0}, 0xc0003ba6c0, 0x0)"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149657395Z level=debug msg="\t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:1722 +0xa36"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149665773Z level=debug msg=google.golang.org/grpc.(*Server).serveStreams.func1.2()
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149674763Z level=debug msg="\t/go/pkg/mod/google.golang.org/grpc@v1.54.0/server.go:966 +0x98"
logger=plugin.grafana-clickhouse-datasource t=2023-07-03T17:41:38.149773423Z level=debug msg="created by google.golang.org/grpc.(*Server).serveStreams.func1"
```

cc @mshustov

### PR DESCRIPTION
**Pull request by aangelisc**
Fixes #440

### CODE REVIEW

File: pkg/plugin/driver_test.go [hunk 1]
```diff
@@ -122,11 +122,21 @@ func TestConnect(t *testing.T) {
...
-		settings := backend.DataSourceInstanceSettings{JSONData: []byte(fmt.Sprintf(`{ "server": "%s", "port": %s, "username": "%s", "secure": %s }`, host, port, username, ssl)), DecryptedSecureJSONData: secure}
+		settings := backend.DataSourceInstanceSettings{JSONData: []byte(fmt.Sprintf(`{ "server": "%s", "port": %s, "username": "%s", "secure": %s, "queryTimeout": "%s"}`, host, port, username, ssl, queryTimeoutString)), DecryptedSecureJSONData: secure}
+		_, err := clickhouse.Connect(settings, json.RawMessage{})
+		assert.Equal(t, nil, err)
+	})
+	t.Run("should not error when valid settings passed - with query timeout as number", func(t *testing.T) {
+		secure := map[string]string{}
+		secure["password"] = password
+		settings := backend.DataSourceInstanceSettings{JSONData: []byte(fmt.Sprintf(`{ "server": "%s", "port": %s, "username": "%s", "secure": %s, "queryTimeout": %d }`, host, port, username, ssl, queryTimeoutNumber)), DecryptedSecureJSONData: secure}
+		fmt.Println(settings)
```
Comment (asimpson) [inline pkg/plugin/driver_test.go:139]: nit: do we need this here?

**Review by asimpson — APPROVED** [review]
Nit aside, looks good to me :trophy:

File: pkg/plugin/driver_test.go (hunk 1 above)
Comment (aangelisc) [reply pkg/plugin/driver_test.go:139]: oops

**Review by adamyeats — APPROVED** [review]
LGTM! 🥇

**Pull request merged**